TrainingApp/Controller/HomeViewController.swift

```diff
@@ -83,9 +159,74 @@ class HomeViewController: UIViewController {
+            Bundle.setLanguage(language)
+        }
+    }
+    
     private func configureUI() {
+        helloLabel.labelSetupForHello()
+        planLabel.labelSetupForPlan()
+        fromLabelRound.labelSetupForFrom()
+        fromLabelOne.labelSetupForFrom()
+        toLabelRound.labelSetupForTo()
+        toLabelOne.labelSetupForTo()
+        departureLabel.labelSetupForDeparture()
+        returnLabelRound.labelSetupForReturn()
+        departureLabelOne.labelSetupForDeparture()
         optionsSegment?.addUnderlineForSelectedSegment()
         optionsSegment?.setFontSize()
+        
+        let roundTitle = NSAttributedString(string: R.string.localizable.roundTrip(), attributes: [NSAttributedString.Key.foregroundColor: R.color.color()!])
+        let onewayTitle = NSAttributedString(string: R.string.localizable.oneWay(), attributes: [NSAttributedString.Key.foregroundColor: R.color.color()!])
+        
+        optionsSegment.setTitle(roundTitle.string, forSegmentAt: 0)
+        optionsSegment.setTitle(onewayTitle.string, forSegmentAt: 1)
+        
+        searchFlightButton.buttonSetupForSearchFlight()
+        spinner.isHidden = true
+    }
+    
+    private func textFieldSetup() {
+        let borderColor = R.color.borderColorBlue()
+        fromTextFieldRound.layer.borderColor = borderColor?.cgColor
+        toTextFieldRound.layer.borderColor = borderColor?.cgColor
+        fromTextFieldOne.layer.borderColor = borderColor?.cgColor
+        toTextField.layer.borderColor = borderColor?.cgColor
+        departureTextField.layer.borderColor = borderColor?.cgColor
+        returnTextField.layer.borderColor = borderColor?.cgColor
+        departureTextFieldOne.layer.borderColor = borderColor?.cgColor
+        
+        fromTextFieldRound.layer.borderWidth = 1
+        toTextFieldRound.layer.borderWidth = 1
+        fromTextFieldOne.layer.borderWidth = 1
+        toTextField.layer.borderWidth = 1
+        departureTextField.layer.borderWidth = 1
+        returnTextField.layer.borderWidth = 1
+        departureTextFieldOne.layer.borderWidth = 1
+    }
+    
+    private func configureNotificationObservers() {
+        fromTextFieldRound.addTarget(self, action: #selector(textDidChange), for: .allEvents)
+        toTextFieldRound.addTarget(self, action: #selector(textDidChange), for: .allEvents)
+        fromTextFieldOne.addTarget(self, action: #selector(textDidChange), for: .allEvents)
+        toTextField.addTarget(self, action: #selector(textDidChange), for: .allEvents)
+    }
+    
+    @objc
+    private func textDidChange(sender: UITextField) {
+        if sender == fromTextFieldRound {
+            viewModel.fromTextField = sender.text
+        } else if sender == toTextFieldRound {
+            viewModel.toTextField = sender.text
+        } else if sender == fromTextFieldOne {
+            viewModel.fromTextField = sender.text
+        } else {
+            viewModel.toTextField = sender.text
+        }
```

Comment:
You can use 

enum sender {
case . fromTextFieldRound
case . toTextFieldRound
case . fromTextFieldOne
case default 
}

Reply:
Do you mean a switch-case statement? If so, I've changed it to a switch-case statement instead of an if-else statement. 👍